components/function-controller/internal/webhook/validating_webhook.go

```diff
@@ -65,9 +67,15 @@ func (w *ValidatingWebHook) handleFunctionValidation(req admission.Request) admi
 			if err := w.decoder.Decode(req, fn); err != nil {
 				return admission.Errored(http.StatusBadRequest, err)
 			}
+
+			if w.rejectJava17AlphaJVM && fn.Spec.Runtime == serverlessv1alpha2.Java17JvmAlpha {
```

Comment:
Nit: I think it's easier to read if we check the runtime first, then check it's enabled/rejected.